examples/ASViewController/Sample/DetailViewController.m

```diff
@@ -19,9 +19,166 @@
 #import <AsyncDisplayKit/AsyncDisplayKit.h>
```

Comment:
Quick not for all of the sample code that changed will get reverted in the final PR. Only here at the moment for testing purposes.